core/sql/optimizer/NAClusterInfo.cpp

```diff
@@ -350,14 +244,8 @@ NAClusterInfo::NAClusterInfo(CollHeap * heap)
         if ((nodeInfo[i].type & MS_Mon_ZoneType_Aggregation) != 0 ||
```

Comment:
Node types are not considered in ComRtUtils.cpp, but it is done here. Is that ok?